src/types/decimal.rs

```diff
@@ -47,6 +49,12 @@ impl Decimal {
         self.coefficient.number_of_decimal_digits()
     }
 
+    /// Returns the Sign of the Decimal value.
+    /// Unscaled zero values may have either a Positive or Negative Sign.
```

Comment:
Couple of thoughts here:
* I believe the phrase "unscaled value" means the same thing as "coefficient" -- is that correct? If so, I'd rather stick to "coefficient" for consistency with the rest of the code base.
* A coefficient of zero will _only_ have a `Negative` sign if it's truly `-0`. I think you may be referring to what will happen if we have to return the coefficient as a `u64`, but that won't impact the `Sign` value which is stored and evaluated separately. (Or have I misunderstood something?)